pytorch_toolkit/instance_segmentation/README.md

```diff
@@ -0,0 +1,263 @@
+# Instance Segmentation in PyTorch
+
+This repository contains inference and training code for Mask R-CNN like
+networks. Models code is designed to enable ONNX export (with custom operations)
+and inference on CPU via OpenVINO.
+[Detectron](https://github.com/facebookresearch/Detectron) and
+[maskrcnn-benchmark](https://github.com/facebookresearch/maskrcnn-benchmark)
+models are wrapped to export their weights to ONNX and OpenVINO.
+
+## Setup
+
+### Prerequisites
+
+* Ubuntu 16.04
+* Python 3.5.2
+* PyTorch 0.4.1
+* OpenVINO 2019 R1 with Python API
+
+### Installation
+
+To install required dependencies run
+
+```bash
+$ cat requirements.txt | xargs -n 1 -L 1 pip3 install
+```
+
+To install the package itself run
+
+```bash
+$ pip3 install -e .
+```
+
+### Get pretrained models
+
+A bunch of top-performing models from Detectron and maskrcnn-benchmark projects
+can be easily obtained and automatically prepared for running in PyTorch and
+OpenVINO, via `tools/download_pretrained_weights.py` script. By default script
+requires no parameters to run, for more details on its configuration run it with
+a `-h` key. This script could also be considered as a sample, showing how models
+are supposed to be converted to ONNX/IR.
+
+### Download MS COCO dataset
+
+To be able to train networks and/or get quality metrics for pre-trained ones,
+one have to download MS COCO dataset. Download [the
+dataset](http://cocodataset.org/#download) (train, val and annotations) and
+unpack it to `./data/coco` folder.
```

Comment:
Will good if you add structure of directory like
```
data/coco/
├── annotations
└── images
    └── val2017
``

Reply:
@AlexanderDokuchaev Structure of the folder was added.